packages/utilities/src/memoize.test.ts

```diff
@@ -98,6 +98,60 @@ describe('memoizeFunction', () => {
     expect(func('a')).toEqual('a4');
     expect(func('a')).toEqual('a4');
   });
+
+  it('caches and preserves falsey values other than null and undefined', () => {
+    let returnFalseyValue = true;
+
+    // tslint:disable-next-line:no-any
+    let falseyValue: any = null;
+
+    let callback = (): number | null => {
+      if (returnFalseyValue) { return falseyValue; }
+      return 1;
+    };
+    let func = memoizeFunction((a: string) => callback(), undefined, true/*ignoreNullOrUndefinedResult */);
+
+    // If falsey value returned by the callback method is null, update the cached value.
+    expect(func('nullFqalseyValue')).toEqual(falseyValue);
```

Comment:
Can you just remove the function parameter if it's not doing anything?

Reply:
Done